flight/mcu/lib/teensyShared/telemetry.cpp

```diff
@@ -0,0 +1,207 @@
+/*
+ * This file defines the telemetry class used to facilitate
+ * telemetry commands and data transfer between the on-board flight
+ * computer and the ground station.
+ *
+ * Spaceshot Avionics 2021-22
+ * Illinois Space Society - Telemetry Team
+ * Gautam Dayal
+ * Nicholas Phillips
+ * Patrick Marschoun
+ * Peter Giannetos
+ */
+
+#include <telemetry.h>
+
+Telemetry::Telemetry() : rf95(RFM95_CS, RFM95_INT) {
+    pinMode(RFM95_RST, OUTPUT);
+    digitalWrite(RFM95_RST, HIGH);
+    delay(100);
+
+    // manual reset
+
+    digitalWrite(RFM95_RST, LOW);
+    delay(10);
+    digitalWrite(RFM95_RST, HIGH);
+    delay(10);
+
+    while (!rf95.init()) {
+        Serial.println("Radio Initialization Failed");
+        while (1)
+            ;
+    }
+    Serial.println("[DEBUG]: Radio Initialized");
+
+    // Defaults after init are 434.0MHz, 13dBm, Bw = 125 kHz, Cr = 4/5, Sf =
+    // 128chips/symbol, CRC on
+
+    if (!rf95.setFrequency(RF95_FREQ)) {
+        Serial.println("[ERROR]: Default setFrequency Failed");
+        while (1)
+            ;
+    }
+
+    /*
```

Comment:
Are we not using this code anymore?

Reply:
that was code to remember the set radio frequency, it didn't have time to be tested before the irec launch